lib/aca_entities/general_agencies/general_agency_account.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module GeneralAgencies
+    class GeneralAgencyAccount < Dry::Struct
+
+      attribute :start_on,                      Types::Date.meta(omittable: false)
+      attribute :end_on,                        Types::Date.optional.meta(omittable: false)
+      attribute :is_active,                     Types::Bool.optional.meta(omittable: false)
+      attribute :aasm_state,                    Types::String.optional.meta(omittable: false)
```

Comment:
The aasm_state should conform to a referenced enumerated type

Reply:
updated.